fishtest/tests/test_rundb.py

```diff
@@ -49,16 +53,15 @@ def test_40_list_LTC(self):
       print(r['args']['tc'])
 
   def test_90_delete_runs(self):
-    run = rundb.get_run(run_id)
-    run['deleted'] = True
-    run['finished'] = True
-    for w in run['tasks']:
-      w['pending'] = False
-    rundb.buffer(run, True)
+    for run in rundb.get_runs():
+      if run['args']['username'] == 'travis' and not 'deleted' in run:
+        print('del ')
+        run['deleted'] = True
+        run['finished'] = True
+        for w in run['tasks']:
+          w['pending'] = False
+        rundb.buffer(run, True)
 
```

Comment:
Should'nt we assert that we have no runs running anymore?

Reply:
update_task will check this, so that should never be a problem